doc/js_tutorials/js_setup/js_setup/js_setup.markdown

```diff
@@ -138,13 +138,6 @@ Building OpenCV.js from Source
     python ./platforms/js/build_js.py build_js --cmake_option="-DOPENCV_EXTRA_MODULES_PATH=opencv_contrib/modules"
     @endcode
 
--#  [optional] To enable OpenCV contrib modules append `--cmake_option="-DOPENCV_EXTRA_MODULES_PATH=/path/to/opencv_contrib/modules/"`
```

Comment:
Nice catch!

relates #19167

Please add `emcmake` prefix to the command above: `emcmake python ...`

Reply:
done!